lib/logstash/inputs/elasticsearch.rb

```diff
@@ -642,6 +640,16 @@ def setup_query_executor
                       end
   end
 
+  def get_transport_client_class
+    require "elasticsearch/transport/transport/http/manticore"
+    require_relative "elasticsearch/patches/_elasticsearch_transport_http_manticore"
```

Comment:
review note: 
- `_elasticsearch_transport_http_manticore` sets 
set user-agent and accept-encoding headers for the es-transport-client versions between 7.2 and 7.16. It seems to me we can remove this but I have kept so far as there are active 7.x users;
- `_elasticsearch_transport_connections_selector` has a round-robin load balancer logic for the es-client versions <7.2.0

I have kept them for now but it looks to me we can remove.